dist/tools/testrunner/testrunner.py

```diff
@@ -19,6 +19,11 @@
 RIOTBASE = os.environ['RIOTBASE'] or \
            os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 
+
+MAKE_TERM_START_WAIT = os.environ.get('TESTRUNNER_START_WAIT') or 3
+MAKE_TERM_START_WAIT = int(MAKE_TERM_START_WAIT)
```

Comment:
Why two lines?

Reply:
For clarity, first step is try to read the config from environment variable.
Second step convert it to the right type.

And I cannot apply `int` directly to os.environ.get in case its None or empty string.

Reply:
Why not

```python
MAKE_TERM_START_WAIT = int(os.environ.get('TESTRUNNER_START_WAIT', 3))
```

?

Reply:
If you do `TESTRUNNER_START_WAIT= make term` it does not use the default value.

Reply:
Ah right, but still: Why not

```python
MAKE_TERM_START_WAIT = int(os.environ.get('TESTRUNNER_START_WAIT') or 3)
```

Reply:
For clarity, each step done on a different line.

Reply:
Mhhhhh... I think it is clear in one line as well, but I don't really care.

Reply:
> If you do TESTRUNNER_START_WAIT= make term it does not use the default value.

I tried this and the default value (3) is used. BTW the command should be
```
TESTRUNNER_START_WAIT= make test
```
I'm with @miri64 and would prefer:
```python
MAKE_TERM_START_WAIT = int(os.environ.get('TESTRUNNER_START_WAIT', 3))
```

Reply:
Hey, the line given is wrong, it should be `MAKE_TERM_START_WAIT = int(os.environ.get('MAKE_TERM_START_WAIT', 3))`

So in this case, you are right, it doesn't work, the command failed because `int()` is invalid. Question: do we really care ?

Reply:
Yes, I care, because I think using `TESTRUNNER_START_WAIT= make test` should use the default value.

Reply:
> Hey, the line given is wrong, it should be MAKE_TERM_START_WAIT = int(os.environ.get('MAKE_TERM_START_WAIT', 3))

@cladmi already addressed that in https://github.com/RIOT-OS/RIOT/pull/8053#discussion_r151383773:

> And I cannot apply int directly to os.environ.get in case its None or empty string.

Reply:
> And I cannot apply int directly to os.environ.get in case its None or empty string.

or any kind of character other than numbers: `MAKE_TERM_START_WAIT=azerty make -C tests/shell/ all test` fails for the same kind of reason as `MAKE_TERM_START_WAIT= make -C tests/shell/ all test`:
```
ValueError: invalid literal for int() with base 10: 'azerty'
```

Reply:
Yes, I care, because I think using TESTRUNNER_START_WAIT= make test should use the default value.

Reply:
> using TESTRUNNER_START_WAIT= make test should use the default value

ok! (sorry)

I still don't like the 2 lines: they could be merged in a single one (and it's still clear I think).

Reply:
If it is that clear, why do you both try to replace the two lines by one wrong one :)
I will add a comment for the first line to explain the case where the variable is defined empty.

Reply:
I was thinking at the second proposition of @miri64 that works with an empty variable:
`MAKE_TERM_START_WAIT = int(os.environ.get('MAKE_TERM_START_WAIT') or 3)`